packages/mobile/src/screens/search-results-screen/tabs/useTrackSearchResultSelect.tsx

```diff
@@ -0,0 +1,24 @@
+import type { ID } from '@audius/common/models'
+import { getSearchBarText } from 'audius-client/src/common/store/search-bar/selectors'
+import { useSelector } from 'react-redux'
+
+import { make, track } from 'app/services/analytics'
+import { EventNames } from 'app/types/analytics'
+
+export const useTrackSearchResultSelect = (
+  kind: 'track' | 'profile' | 'playlist' | 'album'
+) => {
+  const searchQuery: string = useSelector(getSearchBarText)
+  const trackSearchResultSelect = (id: ID) => {
```

Comment:
maybe do a `return useCallback(() => {}...)? you can have a const beforehand, but callback may be good here to ensure rerenders dont propogate a new instance of this callback every time